src/pydata_sphinx_theme/assets/styles/index.scss

```diff
@@ -522,8 +495,44 @@ nav.bd-links {
 }
 
 .bd-sidebar {
+  padding-top: 1em;
```

Comment:
Is this all just moving it around within the `scss` file? AKA no functionality changing?

Reply:
Yup @choldgraf . Just merging two separate definitions of `.bd-sidebar` styles into one.